pages/shop/index.js

```diff
@@ -67,11 +69,30 @@ const Shop = (props) => {
   );
 };
 
+export async function getServerSideProps() {
+  //   context.res.setHeader('Cache-Control', `max-age=${CACHE_MAX_AGE}`);
+
+  const products = await getShopProducts();
+  // console.log(products);
```

Comment:
NIT : Please remove this console statements.

Reply:
Will do